.bazelci/presubmit.yml

```diff
@@ -1,14 +1,45 @@
 ---
+#
+# Bazel releases
+#
+lts: &lts
+  bazel: latest
+
+rolling: &rolling
+  bazel: rolling
+
 tasks:
-  ubuntu1804:
+  ubuntu_lts:
+    name: ubuntu_lts
+    platform: ubuntu2004
+    <<: *lts
     build_targets:
     - "//..."
-  ubuntu2004:
+  ubuntu_rolling:
+    name: ubuntu_rolling
+    platform: ubuntu2004
+    <<: *rolling
     build_targets:
     - "//..."
-  macos:
+  macos_lts:
+    name: macos_lts
+    platform: macos
+    <<: *lts
     build_targets:
     - "//..."
-  windows:
+  windows_lts:
+    name: windows_lts
+    platform: windows
+    <<: *lts
     build_targets:
     - "//..."
+  check_bzlmod:
+    name: check_bzlmod
+    # No need to check bzlmod on every platform. This repository only holds
+    # constants and has no per-platform behavior.
+    platform: ubuntu2004
+    <<: *rolling
```

Comment:
Should this inherit from `ubuntu_rolling` and get the `platform` and `build_targets` from that, instead of redeclaring them?